scripts/eval/eval.py

```diff
@@ -160,123 +159,75 @@ def evaluate_model(
     default_run_name: str = os.environ.get('RUN_NAME', 'llm')
-    run_name: str = pop_config(cfg,
-                               'run_name',
-                               must_exist=False,
-                               default_value=default_run_name)
-    loggers_cfg: Dict[str, Any] = pop_config(cfg,
-                                             'loggers',
-                                             must_exist=False,
-                                             default_value={})
-    eval_subset_num_batches: int = pop_config(cfg,
-                                              'eval_subset_num_batches',
-                                              must_exist=False,
-                                              default_value=-1)
-    icl_subset_num_batches: Optional[int] = pop_config(cfg,
-                                                       'icl_subset_num_batches',
-                                                       must_exist=False,
-                                                       default_value=None)
-    metadata: Optional[Dict[str, str]] = pop_config(cfg,
-                                                    'metadata',
-                                                    must_exist=False,
-                                                    default_value=None,
-                                                    convert=True)
-    should_log_config: bool = pop_config(cfg,
-                                         'log_config',
-                                         must_exist=False,
-                                         default_value=True)
-
-    # Pop out interpolation variables.
-    pop_config(cfg, 'model_name_or_path', must_exist=False, default_value=None)
-    callback_configs: Optional[DictConfig] = pop_config(cfg,
-                                                        'callbacks',
-                                                        must_exist=False,
-                                                        default_value=None)
-
-    # Warn for unused parameters
-    for key in cfg:
-        warnings.warn(
-            f'Unused parameter {key} found in cfg. Please check your yaml to ensure this parameter is necessary.'
-        )
+    run_name = eval_config.run_name if eval_config.run_name else default_run_name
 
-    reproducibility.seed_all(seed)
-    dist.initialize_dist(get_device(None), timeout=dist_timeout)
+    reproducibility.seed_all(eval_config.seed)
+    dist.initialize_dist(get_device(None), timeout=eval_config.dist_timeout)
 
-    if python_log_level is not None:
-        logging.basicConfig(
-            # Example of format string
-            # 2022-06-29 11:22:26,152: rank0[822018][MainThread]: INFO: Message here
-            format=
-            f'%(asctime)s: rank{dist.get_global_rank()}[%(process)d][%(threadName)s]: %(levelname)s: %(name)s: %(message)s'
-        )
-        logging.getLogger('llmfoundry').setLevel(python_log_level.upper())
+    logging.basicConfig(
```

Comment:
should still be behind the if statement?

Reply:
I set the python_log_level to be `debug` by default, so it will never be None, so the if statement is redundant.